composable-tasks/src/compose_tasks.rs

```diff
@@ -161,6 +166,102 @@ pub trait TaskCall:
         EgldOrEsdtTokenPayment::from(payment_out)
     }
 
+    fn smart_swap(
```

Comment:
Why is this function not in `router_actions.rs`?

Reply:
Dependencies issues.

Reply:
Also, smart_swaps uses router now, but may use something else in the future. Maybe even a pair_swap.